geojson_pydantic/base.py

```diff
@@ -53,7 +53,7 @@ def validate_bbox(cls, bbox: Optional[BBox]) -> Optional[BBox]:
         return bbox
 
     @model_serializer(when_used="always", mode="wrap")
-    def clean_model(self, serializer: Any, info: SerializationInfo) -> Dict[str, Any]:
+    def clean_model(self, serializer: Any, info: SerializationInfo) -> Any:
```

Comment:
@markus-work what happens if we use `Dict` or `Dict[Any, Any]` instead of `Any`

TBH, I'm not quite sure of all the implication of this change (cc @eseglem) 

I would also add a comment in the code directly about this change/issue